### ISSUE TITLE
Accessibility - Screen reader would not announce the positions of the toggle buttons

### ISSUE THREAD

**Issue by priyassilect** [highcharts/highcharts#16985]
#### Expected behaviour
The screen reader (Narrator/NVDA) should announce the position of the buttons within the chart. Example: 1 of 3 toggle buttons, 2 of 3 toggle buttons.

#### Actual behaviour
The screen reader just announces the caption and toggle button, it does not announce the positions of the toggle buttons.

#### Live demo with steps to reproduce
In a chart like the one mentioned below, the toggle buttons are used to toggle the visibility of the spline in the chart.  
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/spline-symbols

Enable the Accessibility module. Turn on the screen reader, when you hover or move to the toggle buttons, the screen reader just announces the name of the button and toggling nature. It does not give information about the total buttons in the legend nor the position of the currently selected button. 
Example: 1 of 3 toggle buttons, 2 of 3 toggle buttons. It would be good to have this feature addressed.

#### Product version
Highcharts v9 3.2

#### Affected browser(s)
Edge, Chrome

**Comment by priyassilect** [comment]
Hi,
The Narrator/NVDA both does not announce the positions. It reads the button and the state. In the attached link if you turn on the Narrator and move to the toggle buttons at the legend. It does not say NVDA 1 of 6 buttons, JAWS 2 of 6 buttons etc.

https://jsfiddle.net/oysteinmoseng/cksh7t9e/

**Comment by oysteinmoseng** [comment]
VoiceOver correctly reads these out as 1 of 6, when moving with virtual cursor. Are you by any chance not using the virtual cursor / scan mode?

**Comment by priyassilect** [comment]
I checked again. It seems to read the positions now. Is it something that got updated with the new version?

**Comment by oysteinmoseng** [comment]
Good to hear! I'll mark this as resolved for now then, and we can reopen if needed later.

This structure of `<button>` elements inside `<ul role="list"><li>...` has been out for a few months I believe, but is a relatively recent addition.

**Comment by LucyLiYY** [comment]
The issue is repro on the demo site, could you please reopen this bug? Thanks!

**Comment by oysteinmoseng** [comment]
Reopening to investigate

**Comment by marvin19** [comment]
Hi @LucyLiYY, can you please provide which browser and screen reader you are using?

**Comment by LucyLiYY** [comment]
Browser: Edge
Screen Reader: NVDA
